### PR TITLE
How to create blog posts

### PR DESCRIPTION
**Pull request by sgregg85**
A blog post detailing how to create blog posts as noted in issue #8.

### CODE REVIEW

File: content/2016_10_16_how-to-create-a-blog-post.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+Title: How to create a blog post
```
Comment (aculler) [inline content/2016_10_16_how-to-create-a-blog-post.md]: I would assume that Pelican uses this metadata somewhere when it generates the HTML, so it probably makes sense to title case it.

**Review by aculler — COMMENTED** [review]
Looks good to me with the exception of title casing the title.

**Comment by kjaymiller** [comment]
Great work and I agree on the title casing.

Jay Miller Founder, Host – Productivity in Tech Kjaymiller@gmail.com
On Tue, Oct 18, 2016 at 6:47 AM, Andy Culler notifications@github.com wrote:
@adc4392 commented on this pull request.

Looks good to me with the exception of title casing the title.

In content/2016_10_16_how-to-create-a-blog-post.md [https://github.com/Python-Monthly/Python-Monthly-Website/pull/19#pullrequestreview-4671480] :

> @@ -0,0 +1,48 @@
> +Title: How to create a blog post

I would assume that Pelican uses this metadata somewhere when it generates the HTML, so it probably makes sense to title case it.

**Comment by sgregg85** [comment]
Updated the blog post's title with title case

**Comment by sgregg85** [comment]
Made the changes suggested and @kjaymiller and @adc4392. Going ahead with merge

**Pull request merged**